bftengine/src/bftengine/MsgsCommunicator.hpp

```diff
@@ -34,8 +34,14 @@ class MsgsCommunicator {
   bool isUdp();
 
   [[nodiscard]] bool isMsgsProcessingRunning() const { return incomingMsgsStorage_->isRunning(); }
-  int sendAsyncMessage(bft::communication::NodeNum destNode, char* message, size_t messageLength);
-  void send(std::set<bft::communication::NodeNum> dests, char* message, size_t messageLength);
+  int sendAsyncMessage(bft::communication::NodeNum destNode,
+                       char* message,
+                       size_t messageLength,
+                       bft::communication::NodeNum destEndpointNum);
+  void send(std::set<bft::communication::NodeNum> dests,
+            char* message,
+            size_t messageLength,
+            std::set<bft::communication::NodeNum> destEndpoints);
```

Comment:
Remove the last parameter at all. We don't have the ability to send a message to more than one endpoint (client). In the calls to the communication layer, the default value should be used.

Reply:
done.